### PR TITLE
Update READMEs with make commands

### LINKED ISSUE #450: Update READMEs with new make commands
Now that #443 is merged, need to update all the readmes

### PR DESCRIPTION
**Pull request by chinandrew**
Summary of changes:
- Update template readme with make commands

Going to add these changes to all the readmes, but for reviewing purposes I've just edited the one file so I don't have to go make any reviewer edits to 10+ fies each time.

TODO:
- [x] add updates to all other readmes.

Closes #450

### CODE REVIEW

File: _template_python/README.md [hunk 1]
```diff
@@ -24,34 +24,42 @@ the following:
...
 Unit tests are also included in the module. To execute these, run the following
 command from this directory:
 
 ```
-(cd tests && ../env/bin/pytest --cov=delphi_NAME --cov-report=term-missing)
+make test
 ```
 
+To run individual tests, run the following:
+
```
Comment (krivard) [inline _template_python/README.md:54]: Move the command to occur directly after the colon here, and put the clarification afterwards.

Comment (chinandrew) [reply _template_python/README.md:54]: fixed

**Review by krivard — APPROVED** [review]
Prototype approved!

**Comment by chinandrew** [comment]
readmes updated for other indicators

File: safegraph/README.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
-# Safegraph Mobility Data
+# Patterns Dataset in Safegraph Mobility Data
```
Comment (krivard) [inline safegraph/README.md:1]: This change was probably applied to the wrong safegraph README

File: usafacts/README.md [hunk 3]
```diff
@@ -35,28 +35,61 @@ deactivate
...
+
+If you want to enter the virtual environment in your shell, 
+you can run `source env/bin/activate`. Run `deactivate` to leave the virtual environment. 
+
+Once you are finished, you can remove the virtual environment and 
+params file with the following:
+
+```
+make clean
+```
```
Comment (krivard) [inline usafacts/README.md:68]: I believe this change has accidentally duplicated the "Running the Indicator" section

**Review by krivard — CHANGES_REQUESTED** [review]
Couple flubs to fix

File: safegraph/README.md (hunk 2 above)
Comment (chinandrew) [reply safegraph/README.md:1]: Oops copy pasted and missed this, thanks

File: usafacts/README.md (hunk 3 above)
Comment (chinandrew) [reply usafacts/README.md:68]: thanks for catching, fixed

File: safegraph/README.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
 
-We import raw mobility data from Safegraph, calculate some statistics upon it,
-and aggregate the data from the Census Block Group to County and State levels.
-For detailed information see the files `DETAILS.md` contained
-in this directory.
+We import raw mobility data from Safegraph, calculate some 
+statistics upon it, and aggregate the data from the Zip Code level to County,
+HRR, MSA and State levels. For detailed information see the files `DETAILS.md` 
+contained in this directory.
```
Comment (krivard) [inline safegraph/README.md:6]: Apologies; this change is also bogus; Patterns comes to us as Zip but the vanilla Mobility is CBGs.

Comment (chinandrew) [reply safegraph/README.md]: My bad, I was sloppy doing the copy paste. I saw quidel was pretty much a straight replacement and assumed sg would be as well.

**Review by krivard — APPROVED** [review]
xlnt

**Pull request merged**